README.md

```diff
@@ -3,50 +3,31 @@
 API plugins, microservices, and commands for accessing NetCDF data/metadata contained in iRODS data objects.
 
 Currently supported version(s) of iRODS:
-   - 4.2.8
-
-OS Distributions presently supported:
-   - Centos 7
-   - Ubuntu 16.04
-   - Ubuntu 18.04
+   - 4.3.1
+   - 4.2.12
```

Comment:
Please remove.

The releases page on GitHub covers this.

Reply:
Ok, removed.  Also deleted the line about CMake now being used to build and package, since that's obvious from the text beneath.